rsocket/statemachine/ConsumerBase.cpp

```diff
@@ -64,8 +65,12 @@ void ConsumerBase::processPayload(Payload&& payload, bool onNext) {
     // Frames carry application-level payloads are taken into account when
     // figuring out flow control allowance.
     if (allowance_.tryAcquire()) {
-      sendRequests();
-      consumingSubscriber_->onNext(std::move(payload));
+      VLOG(5) << "ConsumerBase::processPayload(" << onNext << ")";
+      if (consumingSubscriber_) {
```

Comment:
curious. under which circumstances is the consumingSubscriber_ null?

Reply:
@dymk had initially added this to fix a null ptr error.  This PR has gone through a lot of iterations.

Reply:
under which circumstances we get into the nullptr error? We either need to understand this or not add it in the PR. If this is a fix of something else, it should be in a different PR.

Reply:
Acknowledged